### ISSUE TITLE
Blocks Wiki

### ISSUE THREAD

**Issue by Siuolthepic** [QuiltMC/developer-wiki#4]
**What would you like the wiki to cover?**
How to add a block

**Why should this be added to the wiki?**
This game is made up of blocks

**Where in the wiki should this go?**
New section: Blocks

**Comment by Danjhop4** [comment]
yeah it should and include package info so new devs know how to use it thats why i had no luck with fabric

**Comment by ChampionAsh5357** [comment]
Since nobody has claimed this, I'll work on drafting something for this.

**Comment by anonymous123-code** [comment]
Are you still working on this or is this one free again?

**Comment by woodiertexas** [comment]
There's a new blocks page/section on the new wiki fyi: https://modder.wiki.quiltmc.org/blocks/first-block
